s3watcher/lambda/lib/GridApi.ts

```diff
@@ -0,0 +1,104 @@
+import fetch from "node-fetch"
+import { ImportAction, IngestConfig } from "./Lambda"
+import { Logger } from "./Logging"
+
+interface Headers {
+  [name: string]: string
+}
+
+interface Params {
+  [name: string]: string
+}
+
+export interface GridImportRequest {
+  key: string
+  url: string
```

Comment:
Could this be a `URL` type?

Reply:
Wasn't sure about doing this but seems that URL is sane in javascript and fetch already takes it without any call site conversion so 👍